src/main/backend/process.ts

```diff
@@ -147,11 +151,10 @@ export class OwnedBackendProcess implements BaseBackendProcess {
             return;
         }
 
+        await fetch(`${this.url}/shutdown`, { method: 'POST' });
```

Comment:
Could you please go through a `Backend` instance instead? I don't want to have the low-level API fetch logic for `/shutdown` duplicated.
```suggestion
        await getBackend(this.url).shutdown();
```